samples/snippets/requirements.txt

```diff
@@ -1 +1,2 @@
-google-cloud-secret-manager==1.0.0
+#google-cloud-secret-manager==1.0.0
```

Comment:
Nox should be installing the library from source - is that not working correctly?

Reply:
> Nox should be installing the library from source - is that not working correctly?

Thank you for catching this! I did this for debugging locally and forgot to revert it. Now reverted.